fdk/customer_code.py

```diff
@@ -0,0 +1,38 @@
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+from importlib import util
+
+
+class Function(object):
+
+    def __init__(self, func_module_path, entrypoint="handler"):
+        self.spec, self.mod = self.import_from_source(func_module_path)
```

Comment:
Do the we need any exception handling here, or in the other locations below where we interact with the filesystem to load these modules?  What's the expected behavior on failure?